addons/website/models/website_redirect.py

```diff
@@ -13,6 +13,6 @@ class WebsiteRedirect(models.Model):
     redirect_type = fields.Selection([('301', 'Moved permanently (301)'), ('302', 'Moved temporarily (302)')], string='Redirection Type', required=True, default="301")
     url_from = fields.Char('Redirect From', required=True)
     url_to = fields.Char('Redirect To', required=True)
-    website_id = fields.Many2one('website', 'Website')
+    website_id = fields.Many2one('website', ondelete="cascade")
```

Comment:
Why do you remove the string 'Website'? The field then will show "Website_id" as string :S

Reply:
Yikes, I got confused with the 1st arg.

Let me change it, thanks!

Reply:
Not in master. The automatic string is correct, as it removes _id

Reply:
Oh, good to know

Reply:
Oh I didn't know it.

Anyways, since the patch is going to v11+ I prefer to leave it, for easier cherry-pick.

Reply:
Yes, I agree